test/e2e/helm_install_upgrade_test.go

```diff
@@ -65,9 +66,9 @@ func TestHelmUpgrade(t *testing.T) {
 	}{
 		{
 			name:        "upgrade from one before latest to latest minor",
```

Comment:
Actually we don't really need this case. No need to test upgrade between alphas.

Reply:
See: https://github.com/Kong/kong-operator/issues/2158